src/Common/src/System/Net/Logging/EventSourceLogging.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+
+namespace System.Net
+{
+    [EventSource(Name = "Microsoft-System-Net-Debug")]
```

Comment:
It's interesting that this file is in Common, as that implies this will be included in multiple assemblies.  If EventSourceLogging.cs is indeed included in multiple assemblies, doesn't that mean that multiple event sources with the same ID could end up being loaded into the same process?